src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/cap.yaml

```diff
@@ -25,16 +25,16 @@ sylius_fixtures:
                                             en_US:
                                                 name: 'Caps'
                                             fr_FR:
-                                                name: 'Casquettes'
+                                                name: 'Bonnets'
                                         children:
                                             -   code: 'simple_caps'
                                                 translations:
                                                     en_US:
                                                         name: 'Simple'
-                                                        slug: 'caps/simple'
+                                                        slug: 'bonnets/simple'
```

Comment:
We should probably leave the slug for `en_US` translation

Reply:
I agree.

Reply:
Oops, it was a mistake. Fixed!